src/main/java/org/jabref/gui/importer/actions/OpenDatabaseAction.java

```diff
@@ -102,7 +103,7 @@ private Path getInitialDirectory() {
      * @param file the file, may be null or not existing
      */
     public void openFile(Path file, boolean raisePanel) {
-        openFiles(Collections.singletonList(file), raisePanel);
+        openFiles(new ArrayList<>(Collections.singletonList(file)), raisePanel);
```

Comment:
```suggestion
        openFiles(Arrays.asList(file)), raisePanel);
```

Reply:
Yeah I actually first had it like this but I changed it because IntelliJ was complaining to use Collections.singletonList instead of Arrays.asList. Changed it now to reflect your suggestion 👍

Reply:
Yeah, but seems like Intelilj didn't know that you need a mutable list later on.